apps/wallet-mobile/src/migrations/4_26_0.ts

```diff
@@ -0,0 +1,59 @@
+import {App} from '@yoroi/types'
+
+import {WalletMeta} from '../wallet-manager/types'
+import {WALLET_IMPLEMENTATION_REGISTRY} from '../yoroi-wallets/types/other'
+
+export const migrateAddressMode = async (rootStorage: App.Storage) => {
+  const walletsRootStorage = rootStorage.join('wallet/')
+  const addAddressMode = addAddressModeWrapper(walletsRootStorage)
+
+  // moved from /wallet/ -> /
+  await walletsRootStorage.removeItem('deletedWalletIds')
```

Comment:
removed from walletsRoot `/wallet/` -> to `/` app root, so it doesn't conflict with wallet metas.